source/GameInterface/Services/MapEventSides/MapEventSideSync.cs

```diff
@@ -0,0 +1,20 @@
+﻿using GameInterface.AutoSync;
+using HarmonyLib;
+using TaleWorlds.CampaignSystem.MapEvents;
+
+namespace GameInterface.Services.MapEventSides
+{
+    internal class MapEventSideSync : IAutoSync
+    {
+        public MapEventSideSync(IAutoSyncBuilder autoSyncBuilder)
+        {
+            autoSyncBuilder.AddProperty(AccessTools.Property(typeof(MapEventSide), nameof(MapEventSide.CasualtyStrength)));
+            autoSyncBuilder.AddProperty(AccessTools.Property(typeof(MapEventSide), nameof(MapEventSide.LeaderParty)));
+            autoSyncBuilder.AddProperty(AccessTools.Property(typeof(MapEventSide), nameof(MapEventSide.MissionSide)));
+
+
+
+
```

Comment:
Trying to land airplanes here? fix spacing then good to go lol

Reply:
lmao